File: java/8.13/example1.java
import java.util.Random;

public class example1 {
    public static void main(String[] args) {
        Random rand = new Random();
        Integer[] arrayData = new Integer[11];
        for (Integer y = 0; y <= 10; y++) {
            arrayData[y] = rand.nextInt(25565);
        } 
        //output the first 10 elements in the array
        for (Integer count = 0; count < 11; count++) {
            System.out.println(arrayData[count]);
        }
    }
}

File: python/8.13/example1.py
from random import randint # Importing randint from random

arrayData = [] # Setting the value to []

for y in range(1, 11): # Running a loop 10 times
    arrayData += [randint(0, 25565)] # Adding a random number to arrayData

for count in range(0, 10): # Output the first 10 elements in the array
    print(arrayData[count]) # Printing